packages/drivers/routerlicious-driver/src/test/wholeSummaryDocumentStorageService.spec.ts

```diff
@@ -0,0 +1,149 @@
+];
+
+const treeEntries: IWholeFlatSummaryTreeEntry[] = [
+    {
+        path: ".protocol",
+        type: "tree",
+    },
+    {
+        id: "bARCTBK4PQiMLVK2gR5hPRkId",
+        path: ".protocol/attributes",
+        type: "blob",
+    },
+    {
+        id: "bARAL2CXvHYOch_aQtJAJOker",
+        path: ".protocol/quorumValues",
+        type: "blob",
+    },
+    {
+        path: ".app",
+        type: "tree",
+    },
+    {
+        path: ".app/.channels",
+        type: "tree",
+    },
+    {
+        path: ".app/.channels/rootDOId",
+        type: "tree",
+    },
+    {
+        id: "bARCfbIYtOyFwf1+nY75C4UFc",
+        path: ".app/.metadata",
+        type: "blob",
+    },
+];
+
+const flatSummary: IWholeFlatSummary = {
+    id: "bBwAAAAAHAAAA",
+    trees: [
+        {
+            id: "bBwAAAAAHAAAA",
+            sequenceNumber: 0,
+            entries: treeEntries,
+        },
+    ],
+    blobs: summaryBlobs,
+};
+
+const iSummary = {
+    tree: {
+        ".app": {
+            tree: {
+                ".channels": {
+                    tree: {
+                        rootDOId: {
+                            tree: {},
+                            type: 1,
+                            unreferenced: undefined,
+                        },
+                    },
+                    type: 1,
+                    unreferenced: undefined,
+                },
+                ".metadata": {
+                    content: "{\"createContainerRuntimeVersion\":\"0.59.1000\",\"createContainerTimestamp\":1651351060440,\"summaryFormatVersion\":1,\"gcFeature\":0}",
+                    type: 2,
+                },
+            },
+            type: 1,
+            unreferenced: undefined,
+        },
+        ".protocol": {
+            tree: {
+                attributes: {
+                    content: "[]",
+                    type: 2,
+                },
+                quorumValues: {
+                    content: "[[\"code\",{\"key\":\"code\",\"value\":{\"package\":\"no-dynamic-package\",\"config\":{}},\"approvalSequenceNumber\":0,\"commitSequenceNumber\":0,\"sequenceNumber\":0}]]",
+                    type: 2,
+                },
+            },
+            type: 1,
+            unreferenced: undefined,
+        },
+    },
+    type: 1,
+    unreferenced: undefined,
+};
+
+class MockGitManager {
+    public async getSummary(sha: string): Promise<any> {
+        return flatSummary;
+    }
+}
+
+describe("WholeSummaryDocumentStorageService", () => {
+    it("downloads summaries in expected format", async () => {
+        const service = new WholeSummaryDocumentStorageService(
+            "id",
+            new MockGitManager() as any,
+            DebugLogger.create("fluid:testSummaries"),
+        );
+
+        const res = await service.downloadSummary({
+            type: SummaryType.Handle,
+            handleType: SummaryType.Tree,
+            handle: "testHandle",
+        });
+        assert.deepStrictEqual(res, iSummary);
```

Comment:
Can you please add a message to the assert. It helps in debugging and finding out what went wrong when this assert is hit in CI loop.